src/App.tsx

```diff
@@ -1,61 +1,156 @@
+      userId,
+      completed: false,
+      user: getUser(userId),
+    };
+
+    visibleTodos.push(newTodo);
+  };
+
+  const handleChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setIsCorrectTitle(true);
+  };
+
+  const handleChangeUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+(event.target.value));
+    setIsCorrectUser(true);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!userId) {
+      setIsCorrectUser(false);
+    }
+
+    if (!title.trim()) {
+      setIsCorrectTitle(false);
+    }
+
+    if (!userId || !title.trim()) {
+      return;
+    }
+
+    getTodo();
+
+    setUserId(0);
+    setTitle('');
+    setIsCorrectTitle(true);
+    setIsCorrectUser(true);
+  };
+
   return (
-    <div className="App">
-      <h1>Add todo form</h1>
+    <div className="App box">
+      <h1 className="title">Add todo form</h1>
 
-      <form action="/api/users" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
+      <form
+        action="/api/users"
+        method="POST"
+        onSubmit={handleSubmit}
+        className="field"
+      >
+        <div className="field column is-two-fifths">
+          <label className="label">
+            <span>{'Title: '}</span>
+            <input
+              type="text"
+              data-cy="titleInput"
+              className="input is-link "
+              value={title}
+              placeholder="Enter a title"
+              onChange={handleChangeTitle}
+            />
+            {
+              isCorrectTitle
+              || <span className="error">Please enter a title</span>
+            }
+          </label>
         </div>
 
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>Choose a user</option>
-          </select>
+        <div className="field column is-half">
+          <label className="label">
+
+            <p>{'User: '}</p>
+            <div className="select is-norma">
+              <select
+                data-cy="userSelect"
+                defaultValue={userId}
+                value={userId}
+                onChange={handleChangeUser}
+              >
+                <option value="0" disabled>Choose a user</option>
 
-          <span className="error">Please choose a user</span>
+                {
+                  usersFromServer.map(({ name, id }) => (
+                    <option
+                      value={id}
+                      key={id}
+                    >
+                      {name}
+                    </option>
+                  ))
+                }
+              </select>
+            </div>
+          </label>
+
+          {
+            isCorrectUser
+            || <span className="error">Please choose a user</span>
```

Comment:
```suggestion
            hasErrorUser
            && <span className="error">Please choose a user</span>
```
Better use reversed logic and AND operator here (and with title)